alcs-frontend/src/app/features/application/decision/decision-v2/decision-documents/decision-documents.component.ts

```diff
@@ -37,19 +38,21 @@ export class DecisionDocumentsComponent implements OnInit, OnDestroy {
     private decisionService: ApplicationDecisionV2Service,
     private dialog: MatDialog,
     private toastService: ToastService,
-    private confirmationDialogService: ConfirmationDialogService
+    private confirmationDialogService: ConfirmationDialogService,
   ) {}
 
   ngOnInit(): void {
     if (this.decision && !this.loadData) {
       this.dataSource = new MatTableDataSource(this.decision.documents);
-      this.areDocumentsReleased = !this.decision.isDraft && !!this.decision.date && Date.now() >= this.decision.date;
+      this.areDocumentsReleased =
+        !this.hiddenFromPortal && !this.decision.isDraft && !!this.decision.date && Date.now() >= this.decision.date;
```

Comment:
is decision.date just a date or dateTime?

Reply:
Its a number